### PR TITLE
refactor(app-bar): reskin locale and project switchers

### PR DESCRIPTION
**Pull request by emmenko**
image

### CODE REVIEW

File: packages/application-shell/src/components/locale-switcher/locale-switcher.tsx [hunk 1]
```diff
@@ -19,41 +19,47 @@ type Props = {
...
+          flex: 1;
+          color: ${designTokens.colorAccent};
+        `}
+      >
+        {props.children}
+      </span>
+      <span
+        css={css`
+          width: ${themedValue('22px', '26px')};
+          height: ${themedValue('22px', '26px')};
```
Comment (emmenko) [inline packages/application-shell/src/components/locale-switcher/locale-switcher.tsx:45]: Bigger size

File: packages/application-shell/src/components/locale-switcher/locale-switcher.tsx [hunk 2]
```diff
@@ -19,41 +19,47 @@ type Props = {
...
+          border-radius: 100%;
+          background: ${themedValue(
+            designTokens.colorAccent40,
+            designTokens.colorNeutral
+          )};
+          color: ${designTokens.colorSurface};
+          font-size: ${themedValue('0.9rem', '1rem')};
```
Comment (emmenko) [inline packages/application-shell/src/components/locale-switcher/locale-switcher.tsx:52]: Normal font size

File: packages/application-shell/src/components/locale-switcher/locale-switcher.tsx [hunk 3]
```diff
@@ -76,7 +83,10 @@ const LocaleSwitcher = (props: Props) => {
     <div
       css={css`
         position: relative;
-        width: ${designTokens.constraint3};
+        width: ${themedValue(
+          designTokens.constraint3,
+          designTokens.constraint4
+        )};
```
Comment (emmenko) [inline packages/application-shell/src/components/locale-switcher/locale-switcher.tsx:89]: Wider

File: packages/application-shell/src/components/project-switcher/project-switcher.tsx [hunk 4]
```diff
@@ -38,40 +38,46 @@ const PROJECT_SWITCHER_LABEL_ID = 'project-switcher-label';
...
+        `}
+      >
+        <SelectInput.ValueContainer {...restProps}>
+          {restProps.children}
+        </SelectInput.ValueContainer>
+      </div>
+      <span
+        css={css`
+          width: ${themedValue('22px', '26px')};
+          height: ${themedValue('22px', '26px')};
```
Comment (emmenko) [inline packages/application-shell/src/components/project-switcher/project-switcher.tsx:63]: Bigger size

File: packages/application-shell/src/components/project-switcher/project-switcher.tsx [hunk 5]
```diff
@@ -38,40 +38,46 @@ const PROJECT_SWITCHER_LABEL_ID = 'project-switcher-label';
...
+          border-radius: 100%;
+          background: ${themedValue(
+            designTokens.colorAccent40,
+            designTokens.colorNeutral
+          )};
```
Comment (emmenko) [inline packages/application-shell/src/components/project-switcher/project-switcher.tsx:68]: Different background

File: packages/application-shell/src/components/project-switcher/project-switcher.tsx [hunk 6]
```diff
@@ -38,40 +38,46 @@ const PROJECT_SWITCHER_LABEL_ID = 'project-switcher-label';
...
+          color: ${designTokens.colorSurface};
+          font-size: ${themedValue('0.9rem', '1rem')};
```
Comment (emmenko) [inline packages/application-shell/src/components/project-switcher/project-switcher.tsx:70]: Normal font size

**Review by tdeekens — APPROVED** [review]

**Review by kark — APPROVED** [review]

**Review by chloe0592 — APPROVED** [review]

File: packages/application-shell/src/components/locale-switcher/locale-switcher.tsx (hunk 2 above)
Comment (CarlosCortizasCT) [reply packages/application-shell/src/components/locale-switcher/locale-switcher.tsx:52]: Could we maybe use here `ui-kit` tokens?
```suggestion
          font-size: ${themedValue(designTokens.fontSize15, designTokens30)};
```

Comment (emmenko) [reply packages/application-shell/src/components/locale-switcher/locale-switcher.tsx]: Ah thanks! https://github.com/commercetools/merchant-center-application-kit/pull/2993/commits/379af3a8b91f8004f5d44a44a9c7266b53e2051d

**Review by CarlosCortizasCT — APPROVED** [review]
👍

**Comment by FilPob** [comment]
Thanks, looks good

**Review by FilPob — APPROVED** [review]

**Pull request merged**